### DISCUSSION TITLE
0055. 跳跃游戏

### DISCUSSION THREAD

**Discussion by giscus** [itcharge/AlgoNote#105] [Comments]
# 0055. 跳跃游戏

[https://algo.itcharge.cn/solutions/0001-0099/jump-game/](https://algo.itcharge.cn/solutions/0001-0099/jump-game/)

**Comment by yuyichun** [comment]
贪心算法的解法描述里第三步里应该是Maxi >=i,写错了

**Reply by itcharge** [reply]
已修复，感谢指正。

**Comment by Cheng7x** [comment]
循环里的条件个人认为理解为`if max_i >= i: max_i = max(max_i, i + nums[i])`更好懂（

**Reply by itcharge** [reply]
感谢建议，已将文中代码进行优化。
